activities.json

```diff
@@ -166,3 +166,31 @@
 {"activity":"Learn about a distributed version control system such as Git","accessibility":0.0,"type":"education","participants":1,"price":0.0,"link":"","key":"9303608","enabled":true}
 {"activity":"Write a note of appreciation to someone","accessibility":0.0,"type":"social","participants":1,"price":0.0,"link":"","key":"1770521","enabled":true}
 {"activity":"Draw something interesting","accessibility":0.0,"type":"recreational","participants":1,"price":0.0,"link":"","key":"8033599","enabled":true}
+{"activity":"Learn and play a new card game","accessibility":0.0,"type":"recreational","participants":1,"price":0.0,"link":"https://www.pagat.com","key":"9660022","enabled":true}
+{"activity":"Write a list of things you are grateful for","accessibility":0.0,"type":"relaxing","participants":1,"price":0.0,"link":"","key":"2062010","enabled":true}
+{"activity":"Organize one of your collections","accessibility":0.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"6098037","enabled":true}
+{"activity":"Learn Morse code","accessibility":0.0,"type":"education","participants":1,"price":0.0,"link":"https://en.wikipedia.org/wiki/Morse_code","key":"3646173","enabled":true}
+{"activity":"Go for a walk","accessibility":0.1,"type":"relaxation","participants":1,"price":0.0,"link":"","key":"4286250","enabled":true}
+{"activity":"Mow your neighbour's lawn","accessibility":0.2,"type":"charity","participants":1,"price":0.0,"link":"","key":"1303906","enabled":true}
```

Comment:
For consistency, can we use the Americanized spelling of neighbor?

Reply:
Sorry. Habit. Will fix in next commit.